tools/run_tests/run_xds_tests.py

```diff
@@ -1800,9 +1799,8 @@ def __init__(self, compute, alpha_compute, project):
                 rpcs_to_send=rpcs_to_send,
                 metadata_to_send=metadata_to_send)
             logger.debug('running client: %s', client_cmd_formatted)
-            client_cmd = shlex.split(client_cmd_formatted)
             try:
-                client_process = subprocess.Popen(client_cmd,
+                client_process = subprocess.Popen(['/bin/bash', '-i', '-c', client_cmd_formatted],
```

Comment:
Probably best to add a note here with a link to the Node issue, to avoid this getting changed back in the future (I'm not sure, but I think the general recommendation with Popen is to use something like shlex).

Reply:
Done.